bftengine/include/bftengine/ReservedPagesClient.hpp

```diff
@@ -77,6 +77,7 @@ class ResPagesClient : public ReservedPagesClientBase, public IReservedPages {
   }
   /**
    * Should be called at initialization if number of pages is not known at compile time
+   * Call this function in ReplicaForStateTransfer constructor.
```

Comment:
--> "This function must be called only from the ReplicaForStateTransfer constructor before it sets an overall number of reserved pages in the system."

Reply:
The second one variant OK.